### DISCUSSION TITLE
connexion distant

### DISCUSSION THREAD

**Discussion by As1er** [wazuh/wazuh#16431] [Q&A]
Hello everyone,
I installed my wazuh server and now I want to integrate agents who are on another site but I don't know how to do it could you please help me?

**Comment by jctello** [comment]
Hi @As1er , the Wazuh communication protocol is designed to be secure over public networks so in order to integrate agents from a different site you only need to ensure the agents can reach the Wazuh manager over port 1515 for registration and 1514 for communications (in both cases the default protocol is TCP). 

Given that all communications are initiated by the agent, only the manager needs to be open to listen on these ports. 

The agents should then be configured to communicate to an address where they can reach the manager regardless of the site they are in, this may be a public IP or FQDN.

**Reply by As1er** [reply]
Thank you jctello I think that establishing a site-to-site VPN link between the site where my server is located and my annex sites could solve the difficulty too

**Reply by jctello** [reply]
Hi @As1er , indeed Wazuh will be able to work through a VPN or a network-layer proxy, so that is a great approach without having to configure your network to expose the manager to the Internet.
